ietf/.gitignore

```diff
@@ -2,8 +2,8 @@
 /settings_local.py
 /settings_local.py.bak
 /settings_local_debug.py
-/settings_local_sqlitetest.py
 /settings_local_vite.py
 /settings_mysqldb.py
 /settings_postgresqldb.py
+/settings_postgrestest.py
```

Comment:
This file is in git and does not need to be ignored. Please revert this change

Reply:
Reverted by https://github.com/ietf-tools/datatracker/pull/5400/commits/012de38aaee39d4b19928717c151213066527388

Reply:
settings_postgresqldb.py is source-controlled in docker/configs/, and is copied into ietf/ as the dev environment is built - it's runtime configuration for the actual app, not the tests. settings_postgrestest.py is source-controlled in ietf/.

That schism is an artifact of history, and we could normalize what we're doing, but that's a bigger task than this PR should take on.